pkgs/tools/misc/rtz/default.nix

```diff
@@ -27,6 +27,12 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  patches = [
```

Comment:
Shouldn't this be cargoPatches to correctly work?

Reply:
From what I understand, it would be needed to use `cargoPatches` if the patched `Cargo.lock` was used for information on dependency vendoring. Here we have to provide an explicit `cargoLock` anyway because there is a git dependency that needs to be checked, so the regular `patches` works fine.